# The separate mod directory isn't helpful for a EB install since we install a unique version per compiler anyway.
# The pkg-config file can just include one path anyway
# so separating out the (compiler specific) mod files won't help either way.
# Author: mikael.ohman@chalmers.se
--- meson.build.orig	2021-05-10 23:11:21.509151845 +0200
+++ meson.build	2021-05-10 23:32:55.890484048 +0200
@@ -79,17 +79,16 @@
     dftd4_header,
   )
 
-  module_id = meson.project_name() / fc_id + '-' + fc.version()
   meson.add_install_script(
     find_program(files('config'/'install-mod.py')),
-    get_option('includedir') / module_id,
+    get_option('includedir'),
   )
 
   pkg = import('pkgconfig')
   pkg.generate(
     dftd4_lib,
     description: 'Generally Applicable Atomic-Charge Dependent London Dispersion Correction',
-    subdirs: ['', module_id],
+    subdirs: [''],
   )
 
   asciidoc = find_program('asciidoctor', required: false)
